### ISSUE TITLE
Pagination of examples doesn't work

### ISSUE THREAD

**Issue by zakutnyaya** [gradio-app/gradio#812]
Hi guys! We faced the following problem while using **gradio** of **2.8.0 version**.

We use gradio to create some kind of demostand to visualize how our Computer Vision algorithm works.
We upload some sample images (**examples**) to our demostand via the following code (code was simplified):

```
iface = gr.Interface(
    predict,
    gr.inputs.Image(label=‘Photo'),
    [
        gr.outputs.Image(label=‘Prediction')
    ],
    examples=examples,
    examples_per_page=10,
    title="Demostand"
)

iface.launch(
    share=True,
    enable_queue=True,
    server_port=8050,
    server_name='0.0.0.0'
)
```

As you can see, we defined parameter examples_per_page. However, when we run this code, there is no pagination. In other words, all images (~100 items) are uploaded on one page:

<img width="743" alt="Снимок экрана 2022-03-11 в 18 20 13" src="https://user-images.githubusercontent.com/34413164/157987463-f4b83041-1554-4d17-871c-a2d8b321952e.png">

Do you, please, have any idea why pagination of examples doesn’t work in 2.8.0 version?
Thank you in advance!

**Comment by aliabid94** [comment]
Taking a look!

**Comment by abidlabs** [comment]
Closed by #827
